pkg/apis/config/v1alpha3/types.go

```diff
@@ -138,3 +141,11 @@ type PatchJSON6902 struct {
 	// Patch should contain the contents of the json patch as a string
 	Patch string `json:"patch"`
 }
+
+// NodeResources represents the node resources (CPU/Memory)
+type NodeResources struct {
+	// The maximum amount of memory the node can use.
+	Memory string `json:"memory,omitempty"`
```

Comment:
The format should be specified rather than passing directly to a docker flag. Eg in bytes or something.